R/pyramid_chart.R

```diff
@@ -0,0 +1,99 @@
+#' Demography Figure
+#'
+#' @param fig2_data `dataframe` demography data
+#' @param Typev `Factor` Type of Diabetis A or B
+#' @param gendervar `Factor` Gender variable
```

Comment:
Do we want to make this more general? e.g., Factor 1, Factor 2.

Reply:
Updated to add in documentation that the param shows levels of factor.